src/components/button/index.js

```diff
@@ -0,0 +1,13 @@
+export const Button = ({ onClick, children, disabled }) => {
+  return (
+    <button
+      onClick={onClick}
+      className={`text-white py-2 px-4 rounded m-2 ${
+        disabled === true ? "bg-gray-400" : "bg-blue-500"
```

Comment:
no need for disabled === true as it is boolean you can use it direct

Reply:
done